ui/src/pages/CatalogDetails.tsx

```diff
@@ -55,9 +55,11 @@ export default function CatalogDetails() {
           <CatalogSidebar catalog={catalog} />
         </DetailsLayout.Aside>
       </DetailsLayout>
+      {/* NOTE: Due to inconsistencies in the API specification, the following workaround is necessary. 
+          This inconsistency can still be addressed at the interface level. */}
       <EditCatalogDescriptionModal
         open={open}
-        catalog={data}
+        catalog={{ new_name: data.name, ...data }}
```

Comment:
Is `new_name` referring to the comment here? Can we remove `new_name` and just use `comment`